lib/adal.js

```diff
@@ -189,6 +200,28 @@ var AuthenticationContext = (function () {
         // callback from redirected page will receive fragment. It needs to call oauth2Callback
     };
 
+    AuthenticationContext.prototype.loginPopup = function (urlNavigate) {
+        var popupWindow = window.open(urlNavigate, "login", 'width=483, height=600');
```

Comment:
Please do provide a top and left. On a multimon setup, this popped up the window on the other monitor for me.